docs/sql-ref-literals.md

```diff
@@ -71,128 +68,114 @@ SELECT 'it\'s $10.' AS col;
 +---------+
 |It's $10.|
 +---------+
-{% endhighlight %}
+```
 
 ### Binary Literal
 
 A binary literal is used to specify a byte sequence value.
 
 #### Syntax
 
-{% highlight sql %}
+```sql
 X { 'c [ ... ]' | "c [ ... ]" }
-{% endhighlight %}
+```
+
+#### Parameters
 
-#### <em>Parameters</em>
+* **c**
 
-<dl>
-  <dt><code><em>c</em></code></dt>
-  <dd>
     One character from the character set.
```

Comment:
I believe there is limitation on the chars that are allowed in the binary literal ?
for example, i tried :
SELECT X'zzzzzz' AS col and got an exception ?

Reply:
seems to be hexadecimal. Changed to the following: 
```
#### Syntax

```sql
X { 'num [ ... ]' | "num [ ... ]" }
```

#### Parameters

* **num**

    Any hexadecimal number from 0 to F.
```
cc @yaooqinn